listener/upload.go

```diff
@@ -86,24 +88,38 @@ func uploadHandler(event HostEgressEvent) {
 		return
 	}
 
-	err := processUpload(context.Background(), event.Host.ID, event.Host.Account, &event.Host)
+	sys, err := processUpload(event.Host.Account, &event.Host)
 	if err != nil {
 		utils.Log("inventoryID", event.Host.ID, "err", err.Error()).Error(ErrorProcessUpload)
 		messagesReceivedCnt.WithLabelValues(EventUpload, ReceivedErrorProcessing).Inc()
 		return
 	}
 
+	if sys.UnchangedSince != nil && sys.LastEvaluation != nil {
```

Comment:
Could we add test for that? It's not covered in tests and so we are slowly decreasing tests coverage:
https://codecov.io/gh/RedHatInsights/patchman-engine/pull/165/src/listener/upload.go?before=listener/upload.go